.github/workflows/builds.yml

```diff
@@ -285,7 +285,7 @@ jobs:
           CSC_IDENTITY_AUTO_DISCOVERY: false
           SNAPCRAFT_LOGIN: ${{ secrets.SNAPCRAFT_LOGIN }}
         run: |
-          sudo snap install snapcraft --classic
+          sudo snap install snapcraft --channel=5.x/stable --classic
```

Comment:
it should be v6.1 as per this successful build https://github.com/ferdium/ferdium-app/runs/6872922102?check_suite_focus=true#step:16:20